cves/CVE-2015-6826.yml

```
@@ -64,7 +64,15 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  A user could pass a carefully crafted RealMedia file (RealMedia being an
+  audio/video file format) to FFmpeg and, because some fields of a certain
+  struct were not initialized, the user could cause a crash and therefore a
```

```suggestion
  struct were not initialized, the user could cause a crash, leading to a
```
To me this a clearer and more articulate way of phrasing this